roles/tests/test_utils.py

```diff
@@ -0,0 +1,103 @@
+    """
+    Tests for the roles Utils
+    """
+    # pylint: disable=too-many-instance-attributes
+    def __init__(self, *args, **kwargs):
+        super(TestRoleUtils, self).__init__(*args, **kwargs)
+        self.repo_name = 'my little+test repo'
+        self.repo_desc = 'test description'
+        self.repo_slug = slugify(self.repo_name)
+
+        self.group_admin = GroupTypes.REPO_ADMINISTRATOR.format(self.repo_slug)
+        self.group_curator = GroupTypes.REPO_CURATOR.format(self.repo_slug)
+        self.group_author = GroupTypes.REPO_AUTHOR.format(self.repo_slug)
+
+    def check_group_permission(self, repo, group, expected_permissions):
+        """
+        helper function
+        """
+        self.assertListEqual(
+            sorted(get_perms(group, repo)),
+            sorted(expected_permissions)
+        )
+
+    def test_sync_permissions(self):
+        """
+        test the syncpermission
+        """
+        repo = create_repo(self.repo_name, self.repo_desc, self.user.id)
+
+        # permissions before wiping
+        admin = Group.objects.get(name=self.group_admin)
+        self.check_group_permission(
+            repo,
+            admin,
+            RepoPermission.administrator_permissions()
+        )
+        with patch.object(utils, 'roles_init_new_repo') as mock_method:
+            mock_method.return_value = None
+            utils.sync_groups_permissions()
+        # permissions after wiping
+        self.check_group_permission(
+            repo,
+            admin,
+            []
+        )
+        # restore permissions the first call of sync_groups_permissions
+        # will not do anything because the permissions have been alredy wiped
+        utils.sync_groups_permissions()
+        self.check_group_permission(
+            repo,
+            admin,
+            RepoPermission.administrator_permissions()
+        )
+
+    def test_sync_permissions_command(self):
+        """
+        tests the syncpermission via manage.py command
+        """
+        repo = create_repo(self.repo_name, self.repo_desc, self.user.id)
+
+        # permissions before wiping
+        admin = Group.objects.get(name=self.group_admin)
+        self.check_group_permission(
+            repo,
+            admin,
+            RepoPermission.administrator_permissions()
+        )
+        with patch.object(utils, 'roles_init_new_repo') as mock_method:
+            mock_method.return_value = None
+            call_command('sync_permissions')
+        # permissions after wiping
+        self.check_group_permission(
+            repo,
+            admin,
+            []
+        )
+        # restore permissions the first call of sync_groups_permissions
+        # will not do anything because the permissions have been alredy wiped
+        call_command('sync_permissions')
+        self.check_group_permission(
+            repo,
+            admin,
+            RepoPermission.administrator_permissions()
+        )
```

Comment:
You've already tested that with `test_sync_permissions` above, so you really just need a unit test, not an integration test here as it can speed tests up a little bit, but not a big deal.  Happy to merge without changing this.